spec/index.html

```diff
@@ -351,6 +352,64 @@ <h3>Blank nodes</h3>
     </figure>
   </section>
 
+  <section id="section-triple-terms">
+    <h3>Triple terms</h3>
+
+    <p>As the basic unit of description in RDF, the triple is a simple
+    abstraction for stating a directed relationship between two resources.
+    Sometimes, we need to describe more detailed circumstances underlying such
+    relationships. While we could describe these using more sophisticated
+    modelling, there is a tension between simple, direct statements and more
+    complex, granular descriptions. This may require a trade-off between
+    directness and completeness.</p>
+
+    <p>To bridge this difference, we can use <a
+    data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a> as
+    <em>references</em> to simple statements. A triple term is a fixed object
+    that identifies a statement through its constituent subject, predicate and
+    object components.</p>
+
+    <p>This enables us to describe more concrete circumstances, such as events,
+    relationship entities or tokens in data sources, <em>in relation to</em>
+    the simple statements. That allows us to keep them as is, rather than
+    requiring a complete remodelling into more detailed structures, for the
+    sake of occasionally adding details.</p>
+
+    <p>Conceptually, we can <em>annotate</em> a simple statement with a
```

Comment:
Your explanation of "footnotes with details" makes it clearer what your intent was :-)

I can see an analogy with Wikipedia where a claim can be "backed" by multiple _citations_; each of those citations can then have separate provenance data. 
(To me, a footnote provides singular details on a claim; it would rather be in line with the triple term as subject view.)

Reply:
I also believe that annotations are the easiest path to come to triple terms. That was the original RDF-star use case (where embedded triples were also automatically asserted).